community/kernel/src/main/java/org/neo4j/kernel/impl/store/CommonAbstractStore.java

```diff
@@ -649,11 +649,14 @@ public long scanForHighId() throws IOException
             long nextPageId = storeFile.getLastPageId();
```

Comment:
will just comment here, since it's in the same class.
In `rebuildIdGeneratorSlow` long[] freedBatch can by side effect of incorrect read from cursor